### PR TITLE
Rename `_filter` to `choose` and create colspec.jl file in test folder

### PR DESCRIPTION
**Pull request by eliascarv**
This PR renames `_filter` to `choose` and moves ColSpec tests to colspec.jl file.

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -38,6 +38,7 @@ end
 
 # list of tests
 testfiles = [
+  "colspec.jl",
   "distributions.jl",
```
Comment (juliohm) [inline test/runtests.jl:42]: Maybe these two lines should be flipped to follow the same order of the main TableTransforms.jl includes?

**Review by juliohm — COMMENTED** [review]
Great PR as usual.

File: test/runtests.jl (hunk 1 above)
Comment (eliascarv) [reply test/runtests.jl:42]: OK

Comment (eliascarv) [reply test/runtests.jl]: Made in [084a6ef](https://github.com/JuliaML/TableTransforms.jl/pull/41/commits/084a6efea3cbf08d265c257e0b178ef8d545e9b9)

**Comment by juliohm** [comment]
Fix #39

**Pull request merged**